test/__test__/userAdmin.test.js

```diff
@@ -0,0 +1,28 @@
+const request = require('supertest');
+const app = require('../../app');
+const User = require('../../app/models').users;
+const mockUserAdmin = require('../mockUserAdmin');
+const helper = require('../../app/helpers/helpers');
+
+describe('Should get a token', () => {
+  let findOneMock = 0;
+  let adminRoleMock = 0;
+  beforeEach(() => {
+    adminRoleMock = jest.spyOn(helper, 'adminRole').mockResolvedValueOnce(mockUserAdmin.mock1);
+    findOneMock = jest.spyOn(User, 'findOne').mockRejectedValueOnce(mockUserAdmin.mock2);
+  });
```

Comment:
When you are mocking some functions, and you have just one or two tests `beforeEach` is not necessary, you could put those mocks in each test